### ISSUE TITLE
[Manager] Enforce event handler naming convention by linter

### ISSUE THREAD

**Issue by konekoya** [oharastream/ohara#5472]
See #https://github.com/oharastream/ohara/pull/5471#discussion_r457181170

This seems to be a good fit for our usage
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-handler-names.md

**Comment by eechih** [comment]
+1

**Comment by konekoya** [comment]
+1

**Comment by jackyoh** [comment]
ping @konekoya 這個議題可以關閉了嗎？

**Comment by konekoya** [comment]
@jackyoh Nope, this issue links to that PR only because of referencing to it's comment :)

**Comment by konekoya** [comment]
Ping @saivirtue @wu87988622 for suggestion over the topic 🙏

**Comment by konekoya** [comment]
@oharastream/frontend thoughts for this issue? Any suggestion would be welcomed :)
